tasks.star

```diff
@@ -333,7 +320,11 @@ def configure():
                 "mingw64/bin/cmake.exe",
                 "mingw64/bin/SDL2.dll",
             ],
-            cmds = mingw64_bootstrap,
+            cmds = [
+                'usr/bin/bash -lc true',
+                'usr/bin/bash -lc "pacman --noconfirm -Syu"',
+                'usr/bin/bash -lc "pacman --noconfirm -Syu --needed mingw-w64-x86_64-{gcc,xz,ccache,cmake,SDL2} make"',
+            ],
```

Comment:
Doesn't this update MSYS2 a second time on the CI? We can probably drop the `update: true` for `setup-msys2` if we keep this.

Reply:
It actually gets skipped thanks to `skip_if_exists`. Even if it ran it would add maybe 5 seconds for pacman to no-op -- but thanks to your task runner it's a full no-op on CI.

Reply:
Right, I missed that part. It's fine as-is, then.